app/views/components/input_accessible_auto_complete.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.libs.json.Json
+
+@(
+        field: Field,
+        label: Option[String] = None,
+        inputClass: Option[String] = None,
+        formControlClass: Option[String] = None,
+        hint: Option[String] = None,
+        labelClass: Option[String] = None,
+        autoCompleteOptions: Seq[String],
+        autofocus : Boolean = false,
+        useTabIndex : Option[Int] = None,
+        customErrorMessage : Option[String] = None
+)(implicit messages: Messages)
+
+<div class="form-group @if(field.hasErrors){form-group-error}">
+    @if(label.nonEmpty){
+        <label class="form-label" for="@{field.id}">
+            <span class="@if(labelClass.nonEmpty){@labelClass}">@label</span>
+        </label>
+    }
+
+    @if(hint.nonEmpty){
+        <span class="form-hint" id="@{field.id}-hint">@hint</span>
+    }
+
+    @field.errors.map { error =>
+        <span class="error-message" id="@{field.id}-error"><span class="visually-hidden">@messages("error.browser.title.prefix")</span>@customErrorMessage.getOrElse( messages(error.message, error.args: _*))</span>
+    }
+
+    <div id="keyword-search-wrapper" class="autocomplete-wrapper"></div>
+
+
+</div>
+
+<script src="@routes.Assets.at("javascripts/accessible-autocomplete.min.js")" type="text/javascript"></script>
+
+<script type="text/javascript">
```

Comment:
Is it possible to move this into js file?

Reply:
Tried it - script couldn't be moved to a js file as it needs the list of keywords passed form the page into the js function